platform-operator/controllers/verrazzano/installjob/install_config.go

```
@@ -353,6 +372,16 @@ func getInstallArgs(args []installv1alpha1.InstallArgs) []InstallArg {
 	return installArgs
 }
 
+func getRancher(rancher *installv1alpha1.RancherComponent) (Rancher, error) {
```

Looks like an error is never returned here?  If so, you can just have it return *Rancher and inline the calls, to reduce some of the repetitive code.

---

I will fix this

---

Done